jwst/charge_migration/charge_migration.py

```diff
@@ -39,7 +40,8 @@ def charge_migration(input_model, signal_threshold):
     gdq = input_model.groupdq
 
     # Create the output model as a copy of the input
-    output_model = input_model.copy()
+    output_model = input_model
+    input_model.close()
```

Comment:
This (and other places) closes the `input_model` and `output` datamodels (which are the same). The input file can't be closed until after all it's uses are finished (data is used below on line 48). This can lead to errors depending on how the file is used.

Reply:
since the copy is being made at the _step.py level, I removed those lines.